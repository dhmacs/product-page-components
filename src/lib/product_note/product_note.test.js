const productNote = require('./product_note');

test('trivial test', () => {
  expect(true).toBe(true);
});
