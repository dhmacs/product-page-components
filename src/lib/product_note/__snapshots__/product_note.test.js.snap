// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="sc-ifAKCX cqmOwi"
    color="#3B95FF"
  >
    <span
      className="css-1v3opmq"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<svg xmlns=\\"http://www.w3.org/2000/svg\\" width=\\"100%\\" height=\\"100%\\" viewBox=\\"0 0 24 24\\" fill=\\"none\\" stroke=\\"currentColor\\" stroke-width=\\"inherit\\" stroke-linecap=\\"round\\" stroke-linejoin=\\"round\\" class=\\"feather feather-check\\"><polyline points=\\"20 6 9 17 4 12\\"></polyline></svg>",
        }
      }
    />
  </div>,
  <div
    className="sc-bdVaJa bhXfQi"
    fill="#EDF6FF"
  >
    <div
      className="sc-bwzfXH cjdShg"
    >
      <span
        className="sc-htpNat bkojLg"
        color="#085FD9"
      >
        EXAMPLE CONTENTS
      </span>
      <span
        className="sc-bxivhb jsTNBV"
        dangerouslySetInnerHTML={
          Object {
            "__html": "Example Contents",
          }
        }
      />
    </div>
  </div>,
]
`;
